extensions/wikia/SpitfiresContributionExperiments/scripts/challenge.js

```diff
@@ -0,0 +1,138 @@
+require([
+	'jquery',
+	'mw',
+	'ext.wikia.spitfires.experiments.tracker',
+	'wikia.cache',
+	'wikia.loader',
+	'wikia.mustache'
+], function ($, mw, tracker, cache, loader, mustache) {
+	'use strict';
+
+	var experimentName = 'contribution-experiments',
+		experimentId = 5685550912,
+		dismissCookieName = 'contribchallengedismissed';
+
+	function init() {
+		if (
+			mw.user.anonymous() ||
```

Comment:
Yep, this code pre-dated the pre-conditions, it doesn't hurt since it's a super cheap check, but we can remove it.